ios/browser/api/password/brave_password_api.h

```diff
@@ -0,0 +1,123 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_IOS_BROWSER_API_PASSWORD_BRAVE_PASSWORD_API_H_
+#define BRAVE_IOS_BROWSER_API_PASSWORD_BRAVE_PASSWORD_API_H_
+
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+typedef NS_ENUM(NSUInteger, PasswordFormScheme) {
+  PasswordFormSchemeTypeHtml = 0,
+  PasswordFormSchemeTypeBasic,
+  PasswordFormSchemeTypeDigest,
+  PasswordFormSchemeTypeOther,
+  PasswordFormSchemeMobile,
+  PasswordFormSchemeUsernameOnly
+};
+
+@protocol PasswordStoreObserver;
+@protocol PasswordStoreListener;
+
+@class IOSPasswordForm;
```

Comment:
This forward decl is probably not needed

Reply:
Correct, it is removed in https://github.com/brave/brave-core/pull/12360/commits/f1ed2d6425c64f0f9a7d7075c7af10d5d38b04c5